src/registrar/utility/email.py

```diff
@@ -122,6 +144,34 @@ def _can_send_email(to_address, bcc_address):
         if bcc_address and not AllowedEmail.is_allowed_email(bcc_address):
             raise EmailSendingError(message.format(bcc_address))
 
+def get_sendable_addresses(addresses: list[str]) -> tuple[list[str], list[str]]:
+    """Checks whether a list of addresses can be sent to.
+    
+    Returns: a lists of all provided addresses that are ok to send to and a list of addresses that were blocked.
+
+    Paramaters:
+    
+    addresses: a list of strings representing all addresses to be checked.
+    
+    raises:
+        EmailSendingError if email sending is disabled
+    """
+
+    if flag_is_active(None, "disable_email_sending"):  # type: ignore
+        message = "Could not send email. Email sending is disabled due to flag 'disable_email_sending'."
```

Comment:
instead of repeating the message used in _can_send_email, suggest just creating a new type of Error such as EmailingDisabledError that both of these functions use. Also, should it really throw an error in this case or just return two empty lists so that no emails are sent, and log that the flag is on so no addresses are sendable. This may apply to _can_send_email too

Reply:
This is a fair point. I was just copying the logic of _can_send_email, but I think you are correct. I've refactored it, but I left _can_send_email alone for now, let me know if you want that refactored too.